README.md

```diff
@@ -1,6 +1,6 @@
 # login.gov Partner Dashboard
 
-[![Build Status](https://travis-ci.org/18F/identity-dashboard.svg?branch=master)](https://travis-ci.org/18F/identity-dashboard) [![Code Climate](https://codeclimate.com/github/18F/identity-dashboard/badges/gpa.svg)](https://codeclimate.com/github/18F/identity-dashboard) [![Test Coverage](https://codeclimate.com/github/18F/identity-dashboard/badges/coverage.svg)](https://codeclimate.com/github/18F/identity-dashboard/coverage) [![security](https://hakiri.io/github/18F/identity-dashboard/master.svg)](https://hakiri.io/github/18F/identity-dashboard/master)
+[![Build Status](https://travis-ci.org/18F/identity-dashboard.svg?branch=main)](https://travis-ci.org/18F/identity-dashboard) [![Code Climate](https://codeclimate.com/github/18F/identity-dashboard/badges/gpa.svg)](https://codeclimate.com/github/18F/identity-dashboard) [![Test Coverage](https://codeclimate.com/github/18F/identity-dashboard/badges/coverage.svg)](https://codeclimate.com/github/18F/identity-dashboard/coverage) [![security](https://hakiri.io/github/18F/identity-dashboard/main.svg)](https://hakiri.io/github/18F/identity-dashboard/main)
```

Comment:
I don't know who or where to ask about the Hakiri configuration, but it doesn't "just work" to point it to the new branch name.

- Before: https://hakiri.io/github/18F/identity-dashboard/master
- After: https://hakiri.io/github/18F/identity-dashboard/main

Worse comes to worse, we could just remove it?

Reply:
I say remove it? have never used it on my time on the team

Reply:
> I say remove it? have never used it on my time on the team

👍  Removed in 18f2ce5.